### PR TITLE
Add about javascript.info

### PR DESCRIPTION
**Pull request by praveen-me**
{Write down the list of resources or topics you have added.}

{If you have added a resource, please explain your experience with the resource in context of why you have added it.}

--------

### By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted.

- [x] I have read the [Contribution guidelines](https://github.com/aviaryan/learn-for-free/blob/master/CONTRIBUTING.md) and made sure my pull request follows it, specifically the part about promotions and markdown format.
- [x] Any resources I am adding in this PR have been used by me and I can personally vouch for their quality.
- [x] This pull request has a descriptive title. For example, `Add [Resource Name]` or `Add [Topic Name]`, not `Added new entries` or `update`.
- [x] I have double checked that I have only updated the minimum files that are required by this PR.

### CODE REVIEW

File: programming.md [hunk 1]
```diff
@@ -59,3 +59,10 @@ You Don't Know JS is a free series of books for learning JavaScript at a deep le
 Website - https://github.com/getify/You-Dont-Know-JS
 
 [![you don't know js github](https://user-images.githubusercontent.com/4047597/66125034-93c59280-e603-11e9-9821-e7e01c360072.png)](https://github.com/getify/You-Dont-Know-JS)
+
+### JavaScript.Info
+JavaScript.info is a web tutorial where you can find articles related to JavaScript from knob to advanced.
```
Comment (aviaryan) [inline programming.md]: You are missing a space after the heading. 
Also, there seems to be a spelling issue for 'noob'.

File: programming.md [hunk 2]
```diff
@@ -59,3 +59,10 @@ You Don't Know JS is a free series of books for learning JavaScript at a deep le
...
+
+Website: https://javascript.info
+
+![Image of Javascript.info](https://res.cloudinary.com/praveen-me/image/upload/v1570108840/javascript.info.png)
```
Comment (aviaryan) [inline programming.md]: Please link this image to the website as well. Also please host the image on GitHub CDN as it should be more trustable than one's cloudinary.

**Comment by praveen-me** [comment]
Hi @aviaryan, I resolved all feeds.

File: programming.md [hunk 3]
```diff
@@ -59,3 +59,11 @@ You Don't Know JS is a free series of books for learning JavaScript at a deep le
...
 
 [![you don't know js github](https://user-images.githubusercontent.com/4047597/66125034-93c59280-e603-11e9-9821-e7e01c360072.png)](https://github.com/getify/You-Dont-Know-JS)
+
+### JavaScript.Info
+
+JavaScript.info is a web tutorial where you can find articles related to JavaScript from noob to advanced.
+
+Website: https://javascript.info
+
+![javascript.info](https://user-images.githubusercontent.com/29889620/66134989-d5136d80-e616-11e9-8668-cb59e473f57b.png)
```
Comment (aviaryan) [inline programming.md]: The image should link to the website as well. Please see other listings.

Comment (praveen-me) [reply programming.md]: Sorry @aviaryan Finally did this.

Comment (aviaryan) [reply programming.md]: Thank you. Can you check the image URL as well? It seems to be pointing to YDKJS screenshot.

Comment (praveen-me) [reply programming.md]: Yeah, I forget to paste it.
Now, it's fine.

**Comment by aviaryan** [comment]
Thank you @praveen-me. Merged. 🎉

**Pull request merged**